ntc_templates/templates/huawei_smartax_display_ont_info_summary_ont.textfsm

```diff
@@ -1,17 +1,23 @@
+Value Key,Filldown PORT (\S+)
 Value Key ONT_ID (\d+)
-Value RUN_STATE (\w+)
+Value Required RUN_STATE (\w+)
 Value LAST_UPTIME (\S+\s*\S+)
 Value LAST_DOWNTIME (\S+\s*\S+)
 Value LAST_DOWNCAUSE (\S+)
 
 Start
-  ^\s+-
-  ^\s+ONT\s+Run\s+Last\s+Last\s+Last
-  ^\s+ID\s+State\s+UpTime\s+DownTime\s+DownCause -> ONTs
+  ^\s*Command\s*is\s*being\s*executed\.\s*Please\s*wait\s*$$
+  ^\s*-
```

Comment:
This could match a single hyphen with anything after it and accidentally match more lines than intended.

I've updated the rule to allow a hyphen with whitespace on either side, but _not_ other characters.
This will match the horizontal separator lines and is unlikely to match much else.

```suggestion
  ^\s*-+\s*$$
```